src/login.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="style.css">
+    <script src="main.js"></script>
+
+    <title>Trextter - Login</title>
+    <link rel="icon" type="image/x-icon" href="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/160/microsoft/310/t-rex_1f996.png">
+
+</head>
+<body>
+  <form id="login">
```

Comment:
Really liked the login/register pages felt like a nice little easter egg at the end of it all.